MonoGame.Framework.Content.Pipeline/Serialization/Compiler/ContentWriter.cs

```diff
@@ -175,14 +175,6 @@ public void FinalizeContent()
             }
         }
 
-        /// <summary>
-        /// Clears buffer using BinaryWriter's flush method
-        /// </summary>
-        public override void Flush()

```

Comment:
Now that I think about it, removing this will cause an ABI break, people may well be calling this methid in custom content Pipeline extensions.

Reply:
ignore me, the base class has the Flush method.